test/types/virtuals.test.ts

```diff
@@ -1,5 +1,6 @@
 import { Document, Model, Schema, model } from 'mongoose';
 import { expectType } from 'tsd';
+import { autoTypedModel } from './models.test';
```

Comment:
I'd rather avoid tests importing other tests for now. Can we just test the virtuals portion in this file?

Reply:
Sure, will be reacted soon.